charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -126,3 +132,12 @@ kubeletPath: /var/lib/kubelet
 
 ## @param priorityClassName Priority class assigned to daemonset pods
 priorityClassName: ""
+
+restrictedScc:
+  ## @param restrictedScc.enabled Enables the creation of a SecurityContextConstraint based on the restricted SCC with CSI volume support
+  enabled: false
+
+  ## @param restrictedScc.name Set the name of the restricted SCC with CSI support
+  name: ""
+  ## @param restrictedScc.v2 Based up on v2 version of the restricted SCC
+  v2: true
```

Comment:
Is this param required? Would there be a need to not use the v2?

If there is a need to change this, maybe we should do

```
version: v2
```

to allow for future extensibility.

Reply:
This has to do with SCC versioning and specific to OpenShift

Reply:
We can address @marcofranssen suggestion to make it more expandable in the future.